installation/src/Model/ConfigurationModel.php

```diff
@@ -560,6 +560,18 @@ private function createRootUser($options, $db)
 
         try {
             $db->execute();
+
+            // Synch the sequence if pgsql
+            if (($db->getServerType() === 'postgresql') && (!$result)) {
+                $query = $db->getQuery(true)
+                    ->select('MAX(id) +1 as id')
+                    ->from($db->quoteName('#__users'));
+                $db->setQuery($query);
+                $result = $db->loadResult();
+
+                $db->setQuery("SELECT setval('#__users_id_seq', " .  $result . ", false)")
```

Comment:
```suggestion
                $db->setQuery('SELECT setval(' . $db->quote('#__users_id_seq') . ', ' .  $result . ', false)')
```
Use single quotes and `$db->quote`.